main.go

```diff
@@ -3,65 +3,78 @@ package main
 import (
 	"fmt"
 	"os"
-
-	"github.com/aws/aws-sdk-go/aws/ec2metadata"
-	"github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/elb"
 )
 
-const StateInService = "InService"
+func usage() {
+	fmt.Printf("USAGE: %s <check|register|deregister> <load-balancer-name>\n", os.Args[0])
+	os.Exit(2)
+}
+
+func must(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func mustPretty(err error) {
```

Comment:
`must` also prints an stack trace, while `mustPretty` only prints the error message. The reason is that Kubernetes prints the whole output at `kubectl describe`, which is ugly.

But since this is only used once, I can inline this.